test/test_transforms.py

```diff
@@ -153,6 +161,51 @@ def test_to_tensor(self):
         expected_output = ndarray.transpose((2, 0, 1)) / 255.0
         assert np.allclose(output.numpy(), expected_output)
 
+    @unittest.skipIf(accimage is None, 'accimage not available')
+    def test_accimage_to_tensor(self):
+        trans = transforms.ToTensor()
+
+        expected_output = trans(Image.open(GRACE_HOPPER).convert('RGB'))
+        output = trans(accimage.Image(GRACE_HOPPER))
+
+        import visdom
+        vis = visdom.Visdom()
+        vis.image(expected_output.numpy())
+        vis.image(output.numpy())
+        vis.image((expected_output - output).numpy())
+
+        self.assertEqual(expected_output.size(), output.size())
+        assert np.allclose(output.numpy(), expected_output.numpy())
+
+    @unittest.skipIf(accimage is None, 'accimage not available')
+    def test_accimage_resize(self):
+        trans = transforms.Compose([
+            transforms.Scale(256, interpolation=Image.LINEAR),
+            transforms.ToTensor(),
+        ])
+
+        expected_output = trans(Image.open(GRACE_HOPPER).convert('RGB'))
+        output = trans(accimage.Image(GRACE_HOPPER))
+
+        self.assertEqual(expected_output.size(), output.size())
+        self.assertLess(np.abs((expected_output - output).mean()), 1e-3)
+        self.assertLess((expected_output - output).var(), 1e-5)
+        # note the high absolute tolerance
```

Comment:
Yes, individual pixels can be off by a relatively large amount. I adjusted the tolerance so that changing the target size wouldn't trigger an assertion, but using different interpolation methods (bicubic in PIL vs. bilinear in accimage) would trigger an error.